gradle/libs.versions.toml

```diff
@@ -1,18 +1,18 @@
 [versions]
 sonarlint-core = "10.1.0.77731"
 
-sonar-java = "7.33.0.35775"
+sonar-java = "7.34.0.35958"
 sonar-javascript = "10.14.0.26080"
 sonar-php = "3.35.0.11659"
-sonar-python = "4.17.0.14845"
+sonar-python = "4.18.0.15334"
 sonar-kotlin = "2.20.0.4382"
 sonar-slang = "1.15.0.4655"
 sonar-html = "3.16.0.5274"
 sonar-xml = "2.10.0.4108"
-sonar-omnisharp = "1.20.0.89410"
+sonar-omnisharp = "1.21.0.91571"
```

Comment:
I think this version will change with the release, the parent POM has to be updated on sl-omnisharp